### PR TITLE
Awesome book

### PR DESCRIPTION
**Pull request by Uchenna-ibute**
1. Create the navigation bar and design.
2. Make the navigation bar functioning.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,30 +10,71 @@
...
+    <table class="tables">
+      <!--<thead>
+        <tr>
+          <th>Title</th>
+          <th>Author</th>
+          <th></th>
+        </tr>
+      </thead> <br>-->
+      <tbody id="bookli"></tbody>
+    </table>
```
Comment (andersonlebon) [inline index.html]: - [ ] Kindly note that the list of books should only appear when the `list` link in the navbar is clicked. currently, I can see the list of books on all pages. kindly remove the list of books on the contact and form pages

image
 Check this as an :point_right: [example](https://andersonlebon.github.io/Awesome-book/)

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Uchenna-ibute  && @Habtamu210 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: java.js [hunk 2]
```diff
@@ -74,8 +87,31 @@ Form.addEventListener('submit', (e) => {
...
+  let hours = dee.getHours();
+  const ampm = hours >= 12 ? 'pm' : 'am';
+  hours %= 12;
+  const mont = [
+    'january', 'feburary', 'march',
+    'april', 'may', 'june', 'july', 'august', 'september', 'october', 'november', 'december'];
+  const display = `${mont[dee.getMonth()]}
+  ${dee.getDate()}th ${dee.getFullYear()}, ${hours}:${dee.getMinutes()}${ampm}`;
+  timing.innerText = display;
+}
```
Comment (Reem-lab) [inline java.js:112]: - [ OPTIONAL ] I highly recommend making the UI styles here as shown in the given images so please add some more styles for the displyed books to make it looks better and to improve from user experince (UX) 🙏🏻💯🔥🔥


- Your design
image

- Given
image

**Review by Reem-lab — APPROVED** [review]
# Hi Team, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- No Linters error ✅
- Followed GitHub flow ✅
- Great working on the functionlaity ✅
- Professional README ✅🔥

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**